modules/story-budget/story-budget.php

```diff
@@ -133,36 +133,43 @@ function upgrade( $previous_version ) {
 	function action_admin_menu() {
 		add_submenu_page( 'index.php', __('Story Budget', 'edit-flow'), __('Story Budget', 'edit-flow'), apply_filters( 'ef_view_story_budget_cap', 'ef_view_story_budget' ), $this->module->slug, array( $this, 'story_budget') );
 	}
-	
+
 	/**
 	 * Enqueue necessary admin scripts only on the story budget page.
 	 *
 	 * @uses enqueue_admin_script()
 	 */
 	function enqueue_admin_scripts() {
-		global $current_screen;
-		
-		if ( $current_screen->id != self::screen_id )
-			return;
-		
-		$num_columns = $this->get_num_columns();
-		echo '<script type="text/javascript"> var ef_story_budget_number_of_columns="' . esc_js( $this->num_columns ) . '";</script>';
-		
-		$this->enqueue_datepicker_resources();
-		wp_enqueue_script( 'edit_flow-story_budget', $this->module_url . 'lib/story-budget.js', array( 'edit_flow-date_picker' ), EDIT_FLOW_VERSION, true );
+
+		if ( $this->is_story_board_view() ) {
+			$num_columns = $this->get_num_columns();
+			echo '<script type="text/javascript"> var ef_story_budget_number_of_columns="' . esc_js( $this->num_columns ) . '";</script>';
```

Comment:
We should use `wp_json_encode()` (watch for double quotes) instead of `esc_js()`

Reply:
Same as https://github.com/Automattic/Edit-Flow/pull/441/files#r240276539 :)